pumpkin/src/command/commands/cmd_clear.rs

```diff
@@ -101,25 +96,13 @@ impl CommandExecutor for ClearSelfExecutor {
         sender: &mut CommandSender<'a>,
         _server: &crate::server::Server,
         _args: &ConsumedArgs<'a>,
-    ) -> Result<(), InvalidTreeError> {
-        let target = sender
-            .as_player()
-            .ok_or(InvalidTreeError::InvalidRequirementError)?;
+    ) -> Result<(), CommandError> {
+        let target = sender.as_player().ok_or(CommandError::InvalidRequirement)?;
 
-        let items_count = clear_player(&target).await;
+        let item_count = clear_player(&target).await;
 
-        let msg = if items_count == 0 {
-            TextComponent::text_string(format!(
-                "No items were found on player {}",
-                target.gameprofile.name
-            ))
-            .color_named(NamedColor::Red)
-        } else {
-            TextComponent::text_string(format!(
-                "Removed {} item(s) on player {}",
-                items_count, target.gameprofile.name
-            ))
-        };
+        let hold_target = vec![target];
+        let msg = clear_command_text_output(item_count, &hold_target);
 
```

Comment:
Im not sure but could just doing `&[target]` work here instead of allocating a vec?

Reply:
no, rust yelled at me for doing that at first :P

Reply:
Well, that's how it shall be then